test-files/NonDestructiveTest/Expected.osm

```diff
@@ -36090,7 +36090,7 @@
         <tag k="source:commercial_distribution_restriction" v="Unrestricted.  For Public Release."/>
         <tag k="uuid" v="{2EE8AF8A-F80D-47D6-89FF-CFEF440E80E9}"/>
         <tag k="location" v="surface"/>
-        <tag k="hoot:status" v="1"/>
+        <tag k="hoot:status" v="3"/>
```

Comment:
This is 99% of the review.  Ways that were joined back that didn't indicate the right status are now corrected.